.circleci/config.yml

```diff
@@ -412,6 +412,7 @@ jobs:
           message: 'Starting publish for VSCode Extensions'
       - ssh-config
       - checkout
+      - prepare-release-environment-variables
```

Comment:
In the previous release for v49.13.0 there was an issue caused by the previous refactor to prepare-gus-environment-variables. The logic for determining the RELEASE_VERSION was moved into its own command, but wasn't added here for the publish-workflow.

This had two effects:
1. The tag for 49.13.0 wasn't added correctly.
2. The commit message for 'Updated SHA256 v' was missing the version.

Did this have a negative impact on the release itself? - No